virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -360,135 +358,144 @@ include::snippets/technology-preview.adoc[]
+|Displays the virtual machine settings.
+
+|*Name* field
+|Optional: Enter the virtual machine name.
+
+|*SSH key name*
+|Click the edit icon to add a public SSH key to create a secret or to select an existing secret.
 
 |*Start this VirtualMachine after creation* checkbox
 |Clear this checkbox to prevent the virtual machine from starting automatically.
 
-|*Create VirtualMachine*
-|Creates a virtual machine using the boot source and settings chosen on the *InstanceTypes* tab.
+|*Create VirtualMachine* button
+|Creates a virtual machine.
+
+|*YAML & CLI* button
+|Displays the YAML configuration file and the `virtctl create` command to create the virtual machine from the command line.
 |====
 =====
 
-[id="ui-virtualmachines-page"]
+[id="virtualmachines-page_virt-web-console-overview"]
 == VirtualMachines page
 
-You can create and manage VirtualMachines on the VirtualMachines page.
+You create and manage virtual machines on the *VirtualMachines* page.
 
-.VirtualMachines page
+.*VirtualMachines* page
 [%collapsible]
 =====
 [cols="1,3", options="header"]
 |====
 |Element
 |Description
 
-|*Create -> From template*
-|Create a VirtualMachine on the *Catalog* page -> *Template catalog* tab.
-
-|*Create -> From YAML*
-|Create a VirtualMachine by editing a YAML configuration file.
+|*Create* button
+|Create a virtual machine from a template, volume, or YAML configuration file.
 
 |*Filter* field
-|Filter VirtualMachines by status, template, operating system, or node.
+|Filter virtual machines by status, template, operating system, or node.
 
 |Search field
-|Search for VirtualMachines by name or by label.
+|Search for virtual machines by name or by label.
 
-|*VirtualMachines* table
-|List of VirtualMachines
+|Virtual machines table
+|List of virtual machines.
 
-Click the *Options* menu {kebab} beside a VirtualMachine to select *Stop*, *Restart*, *Pause*, *Clone*, *Migrate*, *Copy SSH command*, *Edit labels*, *Edit annotations*, or *Delete*.
+Click the actions menu {kebab} beside a virtual machine to select *Stop*, *Restart*, *Pause*, *Clone*, *Migrate*, *Copy SSH command*, *Edit labels*, *Edit annotations*, or *Delete*.
 
-Click a VirtualMachine to navigate to the VirtualMachine details page.
+Click a virtual machine to navigate to the *VirtualMachine details* page.
 |====
 =====
 
-[id="ui-virtualmachine-details-page"]
+[id="virtualmachine-details-page_virt-web-console-overview"]
 === VirtualMachine details page
 
-You can configure a VirtualMachine on the VirtualMachine details page.
+You configure a virtual machine on the *VirtualMachine details* page.
 
-.VirtualMachine details page
+.*VirtualMachine details* page
 [%collapsible]
 =====
 [cols="1,3", options="header"]
 |====
 |Element
 |Description
 
+|*YAML* switch
+|Set to *ON* to view your live changes in the YAML configuration file. The switch is displayed when you click the *Details* or the *Configuration* tab.
```

Comment:
`s/the *Details* or the *Configuration* tab/the *Details* or *Configuration* tab`